src/components/list/list-base.scss

```diff
@@ -0,0 +1,300 @@
+/** External component calculations */
+$md-checkbox-width: 3 * $baseline-grid !default;
+//TODO(devversion): use shared variable from buttons.scss
```

Comment:
Since it's an issue, which is not depending on that PR, it would make sense to create it.
> See #9196